privacyidea/static/components/config/views/config.policies.details.html

```diff
@@ -308,77 +331,76 @@ <h3 translate>Action</h3>
                                  ng-repeat="actionGroup in actionGroups">
                                 <table class="table table-hover">
                                     <tr ng-repeat="action in actions"
-                                        ng-show="(actionCheckBox[action.name] ||
-                                    !onlySelectedVisible) &&
-                                    (action.name.toLowerCase().match(action_filter.toLowerCase()) !== null ||
-                                    action.desc.toLowerCase().match(action_filter.toLowerCase()) !== null)"
+                                        ng-show="checkOpenGroup(action, action_filter)"
```

Comment:
Where does this come from? 
Is it an internal function - waht does checkOpenGroup do?

Reply:
OK, thank you, found it.